_data/banking.yml

```
@@ -638,6 +792,8 @@ websites:
       - phone
       - sms
     doc: https://www.halifax.co.uk/aboutonline/changes-to-online-banking/
+    regions:
+      - gb
 
   - name: Handelsbanken
```

```suggestion
  - name: Handelsbanken Sverige
```
Handelsbanken also services [Denmark](https://www.handelsbanken.dk/), [Finland](https://www.handelsbanken.fi/), [the Netherlands](https://www.handelsbanken.nl/), [Norway](https://www.handelsbanken.no/), and the [UK](https://www.handelsbanken.co.uk). Each has a different login process and local URL, but we should differentiate the current entry as being in Sweden.